app/models/form526_submission.rb

```diff
@@ -518,6 +535,31 @@ def lighthouse_validation_errors
     end
   end
 
+  def duplicate?
+    last_remediation&.ignored_as_duplicate || false
+  end
+
+  def remediated?
+    last_remediation&.success || false
+  end
+
+  def failure_type?
+    !success_type? && !in_process?
+  end
+
+  def success_type?
+    (submitted_claim_id && backup_submitted_claim_status.nil?) ||
+      (backup_submitted_claim_id && accepted?) || remediated?
+  end
+
+  def in_process?
+    if submitted_claim_id.nil? && backup_submitted_claim_status.nil?
+      created_at >= 1.day.ago
```

Comment:
again, use the scope and check for inclusion

Reply:
Updated in [ccd6694](https://github.com/department-of-veterans-affairs/vets-api/pull/17274/commits/ccd66940f6df82e0f004b5fdce9c60a6bda9be70).